test/regressions/index.test.js

```diff
@@ -101,12 +101,6 @@ async function main() {
         // Move cursor offscreen to not trigger unwanted hover effects.
         page.mouse.move(0, 0);
 
-        // Wait for the flags to load
-        await page.waitForFunction(() => {
-          const images = Array.from(document.querySelectorAll('img'));
```

Comment:
This didn't work, because sometimes the images were not rendered yet - so I moved it after the `'[data-testid="testcase"]:not([aria-busy="true"])'` assertion